### DISCUSSION TITLE
Can I disable scrolling on mouse drag?

### DISCUSSION THREAD

**Discussion by balthild** [sumatrapdfreader/sumatrapdf#2493] [Q&A]
Currently the page will scroll when the left button of my mouse drags on empty areas on the page. I want to use the left button to select texts only, not scrolling the page. Can i achieve this?

**Comment by kjk** [comment]
No, there's no way to achieve that.

**Reply by GitHubRulesOK** [reply]
@balthild 
To select text or image contents within a page area with left mouse
 you need to press CTRL at the same time
then you can right click to copy selection or press a to highlight etc.

**Reply by balthild** [reply]
No, I don't intend to select and copy the images on the page. I am just accustomed to using text selections to highlight those sentences I am reading. This action does not have any problems with some readers (such as Okular), and the left mouse button can select the text nearest to the pointer. However, when using SumatraPDF, dragging with the left mouse button will scroll the page.

**Reply by avagdumortesin** [reply]
I quite honestly don't understand how this is a feature that cannot be turned off for a desktop-only application. This is honestly more of a bug than something that should be marked as "expected product behavior"; there should be an option to turn off this feature, as it complicates a common task with PDF files: selecting text.

**Reply by avagdumortesin** [reply]
Also, if click and drag were meant to be a scrolling option, why is it not mentioned at all in the docs at https://www.sumatrapdfreader.org/docs/Scrolling-and-zooming ?
